common/service/config/config.go

```diff
@@ -135,6 +135,15 @@ type (
 		// A list of paths to files containing the PEM-encoded public key of the Certificate Authorities you wish to trust for client authentication.
 		// This value is ignored if `requireClientAuth` is not enabled.
 		ClientCAFiles []string `yaml:"clientCaFiles"`
+
+		// Optional inline base64 encoded versions of the above file paths
+		// Either specify base64 versions or file paths for all settings.
+		// Do not mix/match both. Set InlineData to true to use base64
+		InlineData   bool     `yaml:"inlineData"`
```

Comment:
I think it'd be better to avoid having a bool flag for this, and instead state that you can't specify both *Data and *File at the same time. The validation code could detect such incorrect combinations, and that would have to be done even with the flag. So, I think the flag is redundant.

Reply:
+1. Removing the flag